### ISSUE TITLE
执行时，内部模块加载pacage.json出错

### ISSUE THREAD

**Issue by mrtone** [pmq20/node-compiler#24]
```
fs.js:583
  return binding.open(pathModule._makeLong(path), stringToFlags(flags), mode);
                 ^

Error: ENOENT: no such file or directory, open '/__enclose_io_memfs__/node_modules/muri/lib/../package.json'
    at Object.fs.openSync (fs.js:583:18)
    at Object.fs.readFileSync (fs.js:490:33)
    at Object.<anonymous> (/__enclose_io_memfs__/node_modules/muri/lib/index.js:240:17)
```

`node_modules/muri/lib/index.js:240:17`
```
/**
 * Version
 */

module.exports.version = JSON.parse(
  require('fs').readFileSync(__dirname + '/../package.json', 'utf8')
).version;
```

**Comment by pmq20** [comment]
Thanks for reporting. I have locally reproduced.

**Comment by pmq20** [comment]
@mrtone Fixed in 

https://github.com/pmq20/libsquash/commit/ec44808a0170edb8c8ff2cd5f337d5f8f317098a

and

https://github.com/pmq20/node-compiler/commit/644b93af64736d3ec14dbd6c27efca9f62810faa

**Comment by pmq20** [comment]
Released in v0.9.5
